bedrock/firefox/templates/firefox/family/index.html

```diff
@@ -4,6 +4,8 @@
 
 {% extends "firefox/base-resp.html" %}
 
+{% add_lang_files "firefox/new/horizon" %}
```

Comment:
I wouldn't do this, file comes with baggage, including activation status and tags.

`Download Firefox` is available in download_button.lang (shared), so we're good there. 

The only new string is `Browse Freely`, we should just add the new string here behind a tag. We can import it from other files for most languages, but using the tag would prevent us from showing the string in English.